R-packages/covidcast/R/plot.R

```diff
@@ -252,6 +253,80 @@ plot_choro = function(x, time_value = NULL, include = c(), range,
   geo = df$geo
   names(val) = geo
 
+  # Make background layer for MSA and HRR maps which are incomplete
+  if ((attributes(x)$metadata$geo_type == "msa") ||
+      (attributes(x)$metadata$geo_type == "hrr")) {
+    map_df = sf::st_read(system.file(
+                         "shapefiles/state",
+                         "cb_2019_us_state_5m.shp",
+                         package = "covidcast"))
+    background_crs = sf::st_crs(map_df)
+    map_df$STATEFP <- as.character(map_df$STATEFP)
+    map_df$is_alaska = map_df$STATEFP == '02'
+    map_df$is_hawaii = map_df$STATEFP == '15'
+    map_df$is_pr = map_df$STATEFP == '72'
+    map_df$STATEFP <- as.numeric(map_df$STATEFP)
+    map_df$is_state = map_df$STATEFP < 57
+    map_df$color = missing_col
+
+    pr_df = map_df %>% filter(.$is_pr)
+    hawaii_df = map_df %>% filter(.$is_hawaii)
+    alaska_df = map_df %>% filter(.$is_alaska)
+
+    main_df = map_df %>% filter(!map_df$is_alaska)
```

Comment:
These 3 lines and probably every other place where you have the same left hand side on 2+ consecutive lines probably can be rewritten using the pipe operator (`%>%`).